torch/quantization/fx/convert.py

```diff
@@ -158,7 +158,9 @@ def convert(model: GraphModule, is_reference: bool = False,
     qconfig_map: Dict[str, QConfigAny] = model._qconfig_map  # type: ignore[assignment]
 
     # move to cpu since we only have quantized cpu kernels
-    model.eval().cpu()
+    model.eval()
+    if not is_reference:
+        model.cpu()
```

Comment:
I would recommend removing this altogether, I don't think quantization code should change the device, this is a job of the user.  Changing it here is unexpected.

Reply:
Ok, will do

Reply:
actually, I can't remove this, looks like some processes like qat are assuming everything coming in is on cpu. Added a todo instead, looks like maybe this should be removed once qat is supported on gpu

Reply:
QAT should be supported on GPU. Even without GPU inference, this is needed.  I will create an issue to track this support.